ee/server/services/account/Account.ts

```diff
@@ -56,12 +57,21 @@ const loginViaResume = async (resume: string): Promise<false | ILoginResult> =>
 	}
 
 	const { when } = user.services?.resume?.loginTokens?.find((token) => token.hashedToken === hashedToken) || {};
+	if (!when) {
```

Comment:
Are you sure this is the right behavior? I thought tokens without when would never expire (e.g. application tokens)

Reply:
I was not, and you're definitely right.. 👍  I'll revert this change